use_cases/transational_templates.md

```diff
@@ -0,0 +1,116 @@
+# Transactional Templates
+
+For this example, we assume you have created a [transactional template](https://sendgrid.com/docs/User_Guide/Transactional_Templates/index.html). Following is the template content we used for testing.
+
+Template ID (replace with your own):
+
+```text
+13b8f94f-bcae-4ec6-b752-70d6cb59f932
+```
+
+Email Subject:
+
+```text
+<%subject%>
+```
+
+Template Body:
+
+```html
+<html>
+<head>
+    <title></title>
+</head>
+<body>
+Hello -name-,
+<br /><br/>
+I'm glad you are trying out the template feature!
+<br /><br/>
+<%body%>
+<br /><br/>
+I hope you are having a great day in -city- :)
+<br /><br/>
+</body>
+</html>
+```
+
+## With Mail Helper Class
+
+```python
+import sendgrid
+import os
+from sendgrid.helpers.mail import Email, Content, Substitution, Mail
+try:
+    # Python 3
+    import urllib.request as urllib
+except ImportError:
+    # Python 2
+    import urllib2 as urllib
+
+sg = sendgrid.SendGridAPIClient(apikey=os.environ.get('SENDGRID_API_KEY'))
+from_email = Email("test@example.com")
+subject = "I'm replacing the subject tag"
+to_email = Email("test@example.com")
+content = Content("text/html", "I'm replacing the <strong>body tag</strong>")
+mail = Mail(from_email, subject, to_email, content)
+mail.personalizations[0].add_substitution(Substitution("-name-", "Example User"))
+mail.personalizations[0].add_substitution(Substitution("-city-", "Denver"))
+mail.template_id = "13b8f94f-bcae-4ec6-b752-70d6cb59f932"
+try:
+    response = sg.client.mail.send.post(request_body=mail.get())
+except urllib.HTTPError as e:
+    print (e.read())
+    exit()
+print(response.status_code)
+print(response.body)
+print(response.headers)
+```
+
+## Without Mail Helper Class
+
+```python
+import sendgrid
+import os
+try:
+    # Python 3
+    import urllib.request as urllib
+except ImportError:
+    # Python 2
+    import urllib2 as urllib
+
+sg = sendgrid.SendGridAPIClient(apikey=os.environ.get('SENDGRID_API_KEY'))
+data = {
+  "personalizations": [
+    {
+      "to": [
+        {
+          "email": "test@example.com"
+        }
+      ],
+      "substitutions": {
+        "-name-": "Example User",
+        "-city-": "Denver"
+      },
+      "subject": "I'm replacing the subject tag"
+    },
+  ],
+  "from": {
+    "email": "test@example.com"
+  },
+  "content": [
+    {
+      "type": "text/html",
+      "value": "I'm replacing the <strong>body tag</strong>"
+    }
+  ],
+  "template_id": "13b8f94f-bcae-4ec6-b752-70d6cb59f932"
+}
```

Comment:
`data` seems to be using 2 space indentation instead of following the rest of the example(s) with 4 space indentation.